openwisp_controller/subnet_division/tasks.py

```diff
@@ -0,0 +1,76 @@
+from celery import shared_task
+from django.utils.translation import ugettext_lazy as _
+from swapper import load_model
+
+Subnet = load_model('openwisp_ipam', 'Subnet')
+IpAddress = load_model('openwisp_ipam', 'IpAddress')
+SubnetDivisionRule = load_model('subnet_division', 'SubnetDivisionRule')
+SubnetDivisionIndex = load_model('subnet_division', 'SubnetDivisionIndex')
+
+
+@shared_task
+def update_subnet_division_index(rule_id):
+    division_rule = SubnetDivisionRule.objects.get(id=rule_id)
```

Comment:
please handle the exceptional case in which the rule_id is not found, it should not happen but in extreme conditions it can happen as it happened with other celery tasks (eg: the worker queue is full and being processed slowly, then the object is deleted by the user and by the time the queue gets to the task the specific ID is not there anymore), see:
https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/tasks.py#L19-L25
Please do it similarly (log a warning if this happens).